packages/atlas/src/components/_crt/HoldersWidget/HoldersWidget.tsx

```diff
@@ -20,7 +20,7 @@ export const HoldersWidget = ({ holders: _holders, ownerId, totalSupply }: Holde
   const [showModal, setShowModal] = useState(false)
 
   const holders = _holders.map((holder) => ({
-    memberId: ownerId,
+    memberId: holder.member.id,
```

Comment:
Unlikely to happen, but still
```suggestion
    memberId: holder?.member?.id || '',
```